main.go

```diff
@@ -35,6 +36,13 @@ func (u *MyUser) GetPrivateKey() crypto.PrivateKey {
 
 func main() {
 
+	myLogger := log.New(os.Stdout, "my logger: ", log.LUTC)
+
+	// make acme lego Logger use our logger
+	legoLog.Logger = myLogger
+
+	log.SetOutput(myLogger.Writer())
```

Comment:
we have our logger we want to use. we can override the lego Logger and make it use our logger. we can also override the "global" logger with `log.SetOutput`.